Download all .fastq files in a project/Download all .fastq files in a project.ipynb

```diff
@@ -0,0 +1,6347 @@
+{
```

Comment:
<p>See above re <code>query</code> variable - should be reformatted for readability</p><p> </p><p><code></code><code></p><p>query = {</p><p>     'query': {</p><p>...</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>